engine/settings/base.py

```diff
@@ -720,5 +721,18 @@ class BrokerTypes:
 PHONE_PROVIDERS = {
     "twilio": "apps.twilioapp.phone_provider.TwilioPhoneProvider",
     # "simple": "apps.phone_notifications.simple_phone_provider.SimplePhoneProvider",
+    "zvonok": "apps.zvonok.phone_provider.ZvonokPhoneProvider",
 }
 PHONE_PROVIDER = os.environ.get("PHONE_PROVIDER", default="twilio")
+
+if PHONE_PROVIDER == "zvonok":
```

Comment:
I believe we don't need this check, because currently we have no way to make it work as expected.

PHONE_PROVIDER value can be changed via UI, but at this point it still  will be twilio, since value is changed through live_settings mechanism and db, not via env.
So, it makes no sense to check value of settings.PHONE_PROVIDER, but live_settings.PHONE_PROVIDER should be checked. However it's impossible, since django  checks settings files before apps are loaded. You can try and will receive:
```django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.``` Exception.

Also, I think that force-disabling GRAFANA_CLOUD_NOTIFICATIONS_ENABLED is not good idea, since someone might want to switch to cloud notifications after zvonok is configured.